client/my-sites/media-library/index.jsx

```diff
@@ -127,17 +130,17 @@ class MediaLibrary extends Component {
 	};
 
 	filterRequiresUpgrade() {
-		const { filter, site, source } = this.props;
+		const { filter, site, source, isJetpack, isAtomic } = this.props;
 		if ( source ) {
 			return false;
 		}
 
 		switch ( filter ) {
 			case 'audio':
-				return ! ( ( site && site.options.upgraded_filetypes_enabled ) || site.jetpack );
+				return ! ( ( site && site.options.upgraded_filetypes_enabled ) || isJetpack );
 
 			case 'videos':
-				return ! ( ( site && site.options.videopress_enabled ) || site.jetpack );
+				return ! ( ( site && site.options.videopress_enabled ) || ( isJetpack && ! isAtomic ) );
```

Comment:
The drop zone isn't loading for me on Atomic with a paid plan 🤔 
<img width="920" alt="Screen Shot 2021-05-26 at 1 34 51 PM" src="https://user-images.githubusercontent.com/1689238/119707546-45dd4e00-be29-11eb-9a3c-8f5979568ee0.png">

Reply:
Nice catch. We are allowing uploading videos for Jetpack sites, whether it's Atomic or not. We'll have to keep working on the conditions.

Reply:
Probably we'll need to check for the `videopress` feature.

Reply:
addressed here https://github.com/Automattic/wp-calypso/pull/53144/commits/13655186c97fb1c2006c82b3733665125db1ff89

Reply:
Looks good, I made a code suggestion here: https://github.com/Automattic/wp-calypso/pull/53144#discussion_r640170700

Reply:
Thanks, accepted and updated.